openstack/maia/templates/_snmp-exporter-f5.yaml.tpl

```diff
@@ -1,6 +1,10 @@
 # WARNING: This file was auto-generated using snmp_exporter generator, manual changes will be lost.
```

Comment:
I guess you hit wrong branch @m-kratochvil :)

Reply:
uh, I thought this .tpl file has to be modified in the prometheus_global branch to federate the new metrics into global Prometheus.. just like you need to modify the grafana_global branch dashboard files to have them available in global grafana.. not the case?

Reply:
It's on the master branch. You can configure the metrics that are federated from the regional Prometheis to the global one [here](https://github.com/sapcc/helm-charts/blob/master/global/prometheus/templates/_prometheus.yaml.tpl#L18) by adding the metric name. Please add as few metrics as possible.

However, we don't federate maia metrics to the global Prometheus.

Reply:
Arno, thanks for the details. The metrics are already present in the file you mentioned on master, but global Prometheus still doesn't offer them. And it offers some other f5 metrics that are not in the file. How did those metrics get there, I'm confused totally.. 
In any case, I guess we can leave this commit here too, it will not hurt anything, correct?

Reply:
Checked the global Prometheus. It needed a force-reload. Should be up-to-date with what is checked in in this rep now. Ping me in slack if you're missing metrics.